python-phoenixdb/dev-support/make_rc.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+############################################################################
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+############################################################################
+set -e
+
+echo "Script that assembles all you need to make an RC."
+echo "It generates source tar in release directory"
+echo "Presumes that you can sign a release as described at https://www.apache.org/dev/release-signing.html"
+echo "Starting...";sleep 2s
+
+# Set directory variables
+DIR_ROOT="$(cd $(dirname $0);pwd)/.."
+echo $DIR_ROOT
+cd $DIR_ROOT
+VERSION=$(cat setup.py | grep '^version = ".*"$' | grep -o '".*"' |sed 's/"//g')
+
+echo $VERSION
```

Comment:
Some labels to accompany these variables (this `echo` and the following 4) would be very appreciated.

Reply:
Those are I just debug echos that I forgot to remove.
I don't think they are needed in production (phoenix core doesn't print them either)